### PR TITLE
Diskspace reclaiming

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

**Comment by dirkmueller** [comment]
(untested)

**Comment by dirkmueller** [comment]
hmm, doesn't work

**Comment by jdsn** [comment]
@dirkmueller nice idea
Is that what we want (if it works)? I thought only the ISOs should be removed.

**Comment by dirkmueller** [comment]
@jdsn yeah, updated.. it seems one has to set -include so that it does anything. testing now.

**Comment by jdsn** [comment]
@dirkmueller btw. PR #2180 also works :)

File: jenkins/ci.suse.de/cloud-mediacheck.yaml [hunk 1]
```diff
@@ -122,3 +117,12 @@
...
           fi
+    publishers:
+      - workspace-cleanup:
+          include:
+            - "solv"
+            - "*.iso"
+          fail-build: false
+          dirmatch: true
+          clean-parent: false
+          external-deletion-command: "echo would delete "
```
Comment (bmwiedemann) [inline jenkins/ci.suse.de/cloud-mediacheck.yaml]: looks like debug code - should that remain active here?

**Comment by dirkmueller** [comment]
I believe it works now

**Pull request merged**